cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -226,33 +226,23 @@ fn register_token() {
 
 #[test]
 fn send_token_to_penpal() {
```

Comment:
This test fails, not sure why yet.

Reply:
Fixed in https://github.com/Snowfork/polkadot-sdk/pull/38/commits/b8bc0172888ce9427f692df000f5afbdd9ca578c

Reply:
Awesome, thanks @yrong!